translations/messages+intl-icu.lt.xlf

```diff
@@ -0,0 +1,430 @@
+            </trans-unit>
+            <trans-unit id="help.show_code">
+                <source>help.show_code</source>
+                <target><![CDATA[Paspauskite mygtuką, kad peržiūrėti <strong>Controller</strong> ir <strong>šabloną</strong> panaudoti atvaizduojant šį puslapį.]]></target>
+            </trans-unit>
+            <trans-unit id="help.browse_app">
+                <source>help.browse_app</source>
+                <target><![CDATA[Naršyti <strong>public dalį</strong> demo puslapio.]]></target>
+            </trans-unit>
+            <trans-unit id="help.browse_admin">
+                <source>help.browse_admin</source>
+                <target><![CDATA[Naršyti <strong>admin dalį</strong> demo puslapio.]]></target>
+            </trans-unit>
+            <trans-unit id="help.login_users">
+                <source>help.login_users</source>
+                <target>išbandykite vieną iš šių vartotojų</target>
+            </trans-unit>
+            <trans-unit id="help.role_user">
+                <source>help.role_user</source>
+                <target>nuolatinis vartotojas</target>
+            </trans-unit>
+            <trans-unit id="help.role_admin">
+                <source>help.role_admin</source>
+                <target>administratorius</target>
+            </trans-unit>
+            <trans-unit id="help.reload_fixtures">
+                <source>help.reload_fixtures</source>
+                <target>Jei vartotojai neveikia, perleiskite šia komandą:</target>
+            </trans-unit>
+            <trans-unit id="help.add_user">
+                <source>help.add_user</source>
+                <target>Jeigu norite sukurti naują vartotoją paleiskite šia komandą:</target>
+            </trans-unit>
+            <trans-unit id="help.more_information">
+                <source>help.more_information</source>
+                <target><![CDATA[Norėdami gauti daugiau informacijos, peržiūrėkite <a href="https://symfony.com/doc">Symfony dokumentciją</a>.]]></target>
+            </trans-unit>
+            <trans-unit id="help.post_summary">
+                <source>help.post_summary</source>
+                <target>Santrauka negali būti su Markdown ar HTML žymom, tiktai paprastas tekstas</target>
+            </trans-unit>
+            <trans-unit id="help.post_publication">
+                <source>help.post_publication</source>
+                <target>Nustatykite datą ateityje, kad suplanuotumėte blogo įrašo paskelbimą.</target>
+            </trans-unit>
+            <trans-unit id="help.post_content">
+                <source>help.post_content</source>
+                <target>Turinį galima formatuoti su Markdown ir HTML žymom.</target>
+            </trans-unit>
+            <trans-unit id="help.comment_content">
+                <source>help.comment_content</source>
+                <target>Komentarai neatitike Elgesio kodekso, bus moderuoti.</target>
```

Comment:
Probably a false positive, but Google Translate says that `neatitike` -> should be -> `neatitinka`